cmd/lakefs/cmd/run.go

```diff
@@ -395,6 +404,25 @@ func checkRepos(ctx context.Context, logger logging.Logger, authMetadataManager
 	}
 }
 
+func deleteExpiredAddressesJob(ctx context.Context, s *gocron.Scheduler, c *catalog.Catalog) error {
+	const deleteExpiredAddressTokensPeriod = 3
+	job, err := s.Every(deleteExpiredAddressTokensPeriod * ref.AddressTokenTime).Do(func() {
+		err := c.DeleteExpiredAddressTokens(ctx)
```

Comment:
Not sure this method should return an error if it returns an error just when it fails to list repositories.
- Change to no value in return and log the error inside the call - like other calls done inside
- Change the function implementation and return multi error to list all the repositories we failed to expire.
I think option 1 is better for this case.

Reply:
done